macros/legacy.asm

```diff
@@ -406,6 +406,9 @@ DEF happinesschecknpc     EQUS "HappinessCheckScript"
 charmap "%", "<BSP>"
 charmap "¯", "<WBR>"
 
+; constants/pokemon_data_constants.asm
+DEF MON_ID EQUS "MON_TRAINER_ID"
+
```

Comment:
Thanks, but we don't need legacy support for *every* change (we rename plenty of functions, structs, etc). It's focused on supporting old map-scripts or music, i.e. things people make without "programming assembly".
```suggestion
```

Reply:
Ah, no worries. Last time I renamed a constant I was asked to add to to the legacy macros, so figured that was something that should regularly be done.